mypyc/ops.py

```diff
@@ -1576,8 +1581,17 @@ def get_method_and_class(self, name: str) -> Optional[Tuple[FuncIR, 'ClassIR']]:
         for ir in self.mro:
             if name in ir.methods:
                 return ir.methods[name], ir
-            if name in ir.properties:
-                return ir.properties[name], ir
+
+            # if name in ir.properties:
+            #     return ir.properties[name][0], ir
+
+            # if name.startswith(PROPSET_PREFIX):
+            #     prop_name = name.split(PROPSET_PREFIX)[1]
+            #     if prop_name in ir.properties:
+            #         setter = ir.properties[prop_name][1]
+            #         if setter:
+            #             return (setter, ir)
```

Comment:
Delete this commented out code. In general, try to clean up this sort of thing before submitting/updating a PR.